src/porepy/params/data.py

```diff
@@ -131,6 +131,56 @@ def set_fluid_compr(self, val):
         self._fluid_compr = val
     fluid_compr = property(get_fluid_compr, set_fluid_compr)
 
+#----------- Fluid density - treated as constant for now
+    def get_fluid_density(self):
+        if hasattr(self, '_fluid_density'):
+            return self._fluid_density
+        else:
+            return 1
```

Comment:
Why not make it a cellwise quantity? At some point in the near future it will be changed anyway.